.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=localhost
```

Comment:
you should totally avoid committing application settings and secret keys to source control.